src/lib.rs

```diff
@@ -48,6 +48,9 @@ mod scalar;
 
 pub use scalar::Scalar;
 
+#[cfg(feature = "basefield")]
+pub use fp::Fp;
```

Comment:
I think it would be a good idea to expose the extensions `Fp2`, `Fp6` and `Fp12` (under this feature or another). They will be needed in applications like `halo2wrong` for example.

Reply:
I think I'll add another feature for them then. And do it in another PR. 
As I think it'll be harder to upstream the extension visibility rather than the basefield only.


On that way we keep that more atomic.